fluent-fallback/src/localization.rs

```diff
@@ -227,474 +116,16 @@ where
     G: BundleGenerator<LocalesIter = P::Iter>,
     P: LocalesProvider,
 {
-    fn get_bundles(&self) -> &Bundles<G> {
-        self.bundles.get_or_init(|| {
-            if self.sync {
-                Bundles::Iter(Cache::new(
-                    self.generator
-                        .bundles_iter(self.provider.locales(), self.res_ids.clone()),
-                ))
-            } else {
-                Bundles::Stream(AsyncCache::new(
-                    self.generator
-                        .bundles_stream(self.provider.locales(), self.res_ids.clone()),
+    pub fn bundles(&self) -> Rc<Bundles<G>> {
```

Comment:
May be worth returning a reference and let the caller clone if needed in order to save some ref-churn specially for sync cases, your call.